test/activerecord_spanner_adapter/information_schema_test.rb

```diff
@@ -90,8 +90,8 @@ def test_list_all_tables
     assert_equal result.length, 1
 
     assert_sql_equal(
-      last_executed_sql,
-      "SELECT * FROM information_schema.tables WHERE table_schema=''"
+      "SELECT TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, PARENT_TABLE_NAME, ON_DELETE_ACTION FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA=''",
```

Comment:
Many of these assertions were the wrong way around (actual before expected, while the function specifies expected before actual).